ui/src/shared/components/LineGraph.js

```diff
@@ -193,6 +195,26 @@ export default React.createClass({
     )
   },
 
+  yLabelClass() {
```

Comment:
interesting. i wonder with this and `getLabel` if there's a less DOM-dependent way to do this, but it's not coming to mind for me right now either.